float_data_for_tock.py

```diff
@@ -0,0 +1,60 @@
+import requests
+import os
+import json
+
+api_key = os.environ.get('FLOAT_API_KEY')
+headers = {'Authorization': 'Bearer ' + api_key}
+endpoint = 'tasks'
+year = "2016"
+month = "04"
+day = "01"
+weeks = "52"
+timecard_date = str(year + "-" + month + "-" + day)
+payload = {'start_day': timecard_date, 'weeks': weeks}
+url = 'https://api.floatschedule.com/api/v1/' + endpoint
+r = requests.get(url, headers=headers, params=payload)
+output = r.content
+
+print "\nData successfully pulled from Float API. \n"
+
+fmt = 'json'
+def writeToFile(x, str_data={}, fmt=fmt):
+    with open("output." + fmt,"w") as fp:
+        fp.write(x)
+
+writeToFile(output)
```

Comment:
this extra step (writing the data to file, then reopening) is the only way i could get the api output to cooperate. assuming this is a less involved way of addressing.